bin/tact

```diff
@@ -766,12 +766,11 @@ EOF
 		    end
 		  end
                 }
-	      }
-  
-              inner_threads.each { |aThread|
+
+                inner_threads.each { |aThread|
                 aThread.join
+                }
```

Comment:
The moving of synchronization block is unclear. If we do so, it means all baselines perform one-by-one, therefore, can delete inner_threads at all.
Do you have a test case when parallel baseline inner threads interfere?

Reply:
Thank you for your review.
Suppose such a scenario:
    <baseline description="-O2" flags="-O2" />
    <baseline description="-O3" flags="-O3" />
    <baseline description="-Os" flags="-Os" />

The 3 TestRunner may be executed concurrently on the same board：make clean, make and run.
Maybe when TestRunner1 executes make, TestRunner2 is executing make clean.
Moreover, several TestRunners execute the test program on the same board at the same time, and the test data collected may be affected.
Further, when each TestRunner executes, it should first set the CPU frequency, clear the memory cache, close the unrelated program, and so on.
Thanks.